draft-ietf-oauth-status-list.md

```diff
@@ -818,6 +818,8 @@ Resulting in the byte array and compressed/base64url-encoded Status List:
 
 # Security Considerations {#Security}
 
+The Status List as defined in this specification only exists in cryptographically secured containers which allows checking the integrity and origin without relying on other aspects like transport security (e.g., the web PKI).
```

Comment:
this is a little confusing to me, are we talking about the Status List object, that is only communciated in Status List Tokens or about referenced Tokens?

Reply:
status list object currently (after the change) only exists in secured containers

Reply:
should we add reference to the section?

Reply:
```suggestion
The Status List as defined in [](#status-list) only exists in cryptographically secured containers which allows checking the integrity and origin without relying on other aspects like transport security (e.g., the web PKI).
```